src/Control/Comonad/Trans/Env.hs

```diff
@@ -72,6 +72,9 @@ import Control.Comonad.Trans.Class
 import Data.Foldable
 import Data.Traversable
 #endif
+#if MIN_VERSION_base(4,9,0)
+import Data.Functor.Classes
```

Comment:
For `comonad`, I think one should depend on `transformers` and `transformers-compat` to get these instances across the board.